### PR TITLE
Add CMS BenefitsPage

### LINKED ISSUE #4429: Alumni benefits page
As a learner, I'd like to have a page that explains all the benefits from the MIT alumni association for people who earn a program certificate. 

As a marketing team member, I'd like to have a single URL that I can use in marketing to all potential users with a description of the benefit.

#### Designs and Mockups

- [x] Design is TBD

#### Acceptance Criteria:

- [x] New page in CMS, child of the home page and editable by marketing and the MicroMasters team
- [x] link to Alumni association

### PR DESCRIPTION
**Pull request by annagav**
#### What are the relevant tickets?
Fix #4429

#### What's this PR do?
Add a cms page BenefitsPage

#### How should this be manually tested?
run migrations, login into /cms/, under HomePage create a BenefitsPage, fill out the page and preview it.

Screen Shot 2019-10-21 at 1 46 55 PM

### CODE REVIEW

**Comment by pdpinch** [comment]
Thanks @annagav. How do we want to approach the home page changes, including the link to this page? Should that be a separate PR, or part of this one?

**Comment by annagav** [comment]
@pdpinch For the home page changes I will open a separate PR, but it is not very clear to me how and where to place the link to the benefits page. I was thinking we could put it next to the title of the section labeled as "Why should I enroll?".

**Comment by abdulkdawson** [comment]
@annagav @pdpinch I like the hero image but for this page we should crop it to about 1/4 of the current size. An image this size seems unnecessary for this type of page.

**Comment by annagav** [comment]
@pdpinch @abdulkdawson Here is a reduced size background image:

Screen Shot 2019-10-24 at 8 17 01 AM

**Comment by abdulkdawson** [comment]
@annagav this image size is much better.

**Comment by annagav** [comment]
I addressed Abdul's comments, now it is ready for full review.

File: cms/models.py [hunk 1]
```diff
@@ -72,6 +71,59 @@ def get_program_page(program):
...
+
+        context["is_public"] = True
+        context["has_zendesk_widget"] = True
+        context["google_maps_api"] = False
+        context["authenticated"] = not request.user.is_anonymous
+        context["is_staff"] = has_role(request.user, [Staff.ROLE_ID, Instructor.ROLE_ID])
+        context["username"] = username
+        context["js_settings_json"] = json.dumps(js_settings)
+        context["title"] = self.title
+        context["ga_tracking_id"] = ""
```
Comment (tasawernawaz) [inline cms/models.py]: I am not sure, if all of there values are necessary for the context.

**Comment by tasawernawaz** [comment]
It looks like there are some flaky tests, after rerunning the build it has passed now.

File: cms/models.py (hunk 1 above)
Comment (annagav) [reply cms/models.py]: These values are used in base.html and other included templates.

**Pull request merged**